# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.h
# Commit: 9213b6bda67a
# Full Hash: 9213b6bda67a689644f442f7b98902c5a5184c74
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2021-09-24 09:46:13
# Regressor Bug: 1725572
# File Overlap Count: 1
# Description:
#   Bug 1725572 - Part 2: Keep BrowsingContext alive until every process has acknowledged the discard, r=farre
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D125897
# ==============================================================================

diff -r 6bb38c7309e5 -r 9213b6bda67a dom/ipc/ContentChild.h
--- a/dom/ipc/ContentChild.h	Thu Sep 23 18:54:31 2021 +0000
+++ b/dom/ipc/ContentChild.h	Thu Sep 23 18:54:31 2021 +0000
@@ -686,7 +686,7 @@
       uint64_t aGroupId, BrowsingContext::IPCInitializer&& aInit);
 
   mozilla::ipc::IPCResult RecvDiscardBrowsingContext(
-      const MaybeDiscarded<BrowsingContext>& aContext,
+      const MaybeDiscarded<BrowsingContext>& aContext, bool aDoDiscard,
       DiscardBrowsingContextResolver&& aResolve);
 
   mozilla::ipc::IPCResult RecvRegisterBrowsingContextGroup(